pom.xml

```diff
@@ -110,6 +111,11 @@
 				</exclusions>
 			</dependency>
 			<dependency>
+				<groupId>org.springframework.integration</groupId>
```

Comment:
@ilayaperumalg sure thing - i pulled it from the parent and put it in the module itself.